packages/core/integration-tests/test/webmanifest.js

```diff
@@ -0,0 +1,46 @@
+const assert = require('assert');
+const path = require('path');
+const {
+  bundle,
+  assertBundles,
+  distDir,
+  outputFS
+} = require('@parcel/test-utils');
+
+describe('webmanifest', function() {
+  it('should support webmanifest', async function() {
+    let b = await bundle(
+      path.join(__dirname, '/integration/webmanifest/index.html')
+    );
+
+    await assertBundles(b, [
+      {
+        name: 'index.html',
+        assets: ['index.html']
+      },
+      {
+        type: 'webmanifest',
+        assets: ['manifest.webmanifest']
+      },
+      {
+        type: 'png',
+        assets: ['icon.png']
+      },
+      {
+        type: 'png',
+        assets: ['screenshot.png']
+      },
+      {
+        type: 'js',
+        assets: ['serviceworker.js']
+      }
+    ]);
+  });
+
+  it("should treat webmanifest as an entry module so it doesn't get content hashed", async function() {
```

Comment:
This test passes, but when I check `dist` dir there is no `index.html`.

btw, how can I get bundling result after particular test case?

Reply:
By default, the tests write to an in-memory filesystem rather than the real file system. If you want, you can change the `outputFS` to a `NodeFS` instance temporarily if you need to test something. https://github.com/parcel-bundler/parcel/blob/v2/packages/core/test-utils/src/utils.js#L27